--- origsrc/pycairo-1.8.10/src/context.c	2010-09-11 10:15:34.000000000 -0500
+++ src/pycairo-1.8.10/src/context.c	2010-11-03 15:41:38.907391100 -0500
@@ -1356,7 +1356,7 @@ static PyMethodDef pycairo_methods[] = {
 };
 
 PyTypeObject PycairoContext_Type = {
-  PyVarObject_HEAD_INIT(&PyType_Type, 0)
+  PyVarObject_HEAD_INIT(NULL, 0)
   //  PyObject_HEAD_INIT(NULL)
   //0,                                  /* ob_size */
   "cairo.Context",                    /* tp_name */
--- origsrc/pycairo-1.8.10/src/font.c	2010-09-11 10:16:05.000000000 -0500
+++ src/pycairo-1.8.10/src/font.c	2010-11-03 15:41:38.922991100 -0500
@@ -90,7 +90,7 @@ static PyMethodDef font_face_methods[] =
 */
 
 PyTypeObject PycairoFontFace_Type = {
-  PyVarObject_HEAD_INIT(&PyType_Type, 0)
+  PyVarObject_HEAD_INIT(NULL, 0)
   //PyObject_HEAD_INIT(NULL)
   //0,                                  /* ob_size */
   "cairo.FontFace",                   /* tp_name */
@@ -185,7 +185,7 @@ static PyMethodDef toy_font_face_methods
 
 
 PyTypeObject PycairoToyFontFace_Type = {
-  PyVarObject_HEAD_INIT(&PyType_Type, 0)
+  PyVarObject_HEAD_INIT(NULL, 0)
   //PyObject_HEAD_INIT(NULL)
   //0,                                  /* ob_size */
   "cairo.ToyFontFace",                /* tp_name */
@@ -354,7 +354,7 @@ static PyMethodDef scaled_font_methods[]
 
 
 PyTypeObject PycairoScaledFont_Type = {
-  PyVarObject_HEAD_INIT(&PyType_Type, 0)
+  PyVarObject_HEAD_INIT(NULL, 0)
   //PyObject_HEAD_INIT(NULL)
   //0,                                  /* ob_size */
   "cairo.ScaledFont",                 /* tp_name */
@@ -541,7 +541,7 @@ static PyMethodDef font_options_methods[
 
 
 PyTypeObject PycairoFontOptions_Type = {
-  PyVarObject_HEAD_INIT(&PyType_Type, 0)
+  PyVarObject_HEAD_INIT(NULL, 0)
   //PyObject_HEAD_INIT(NULL)
   //0,                                  /* ob_size */
   "cairo.FontOptions",                /* tp_name */
--- origsrc/pycairo-1.8.10/src/matrix.c	2010-09-11 09:13:19.000000000 -0500
+++ src/pycairo-1.8.10/src/matrix.c	2010-11-03 15:41:38.938591100 -0500
@@ -284,7 +284,7 @@ static PyMethodDef matrix_methods[] = {
 };
 
 PyTypeObject PycairoMatrix_Type = {
-  PyVarObject_HEAD_INIT(&PyType_Type, 0)
+  PyVarObject_HEAD_INIT(NULL, 0)
   //PyObject_HEAD_INIT(NULL)
   //0,                                  /* ob_size */
   "cairo.Matrix",                     /* tp_name */
--- origsrc/pycairo-1.8.10/src/path.c	2010-09-11 09:06:57.000000000 -0500
+++ src/pycairo-1.8.10/src/path.c	2010-11-03 15:41:38.938591100 -0500
@@ -165,7 +165,7 @@ static PyObject * path_iter(PyObject *se
 
 
 PyTypeObject PycairoPath_Type = {
-  PyVarObject_HEAD_INIT(&PyType_Type, 0)
+  PyVarObject_HEAD_INIT(NULL, 0)
   //PyObject_HEAD_INIT(NULL)
   //0,				        /* ob_size */
   "cairo.Path",			/* tp_name */
@@ -286,7 +286,7 @@ pathiter_next(PycairoPathiter *it) {
 }
 
 PyTypeObject PycairoPathiter_Type = {
-  PyVarObject_HEAD_INIT(&PyType_Type, 0)
+  PyVarObject_HEAD_INIT(NULL, 0)
   //PyObject_HEAD_INIT(NULL)
   //0,                                  /* ob_size */
   "cairo.Pathiter",                   /* tp_name */
--- origsrc/pycairo-1.8.10/src/pattern.c	2010-09-11 07:35:31.000000000 -0500
+++ src/pycairo-1.8.10/src/pattern.c	2010-11-03 15:41:38.954191100 -0500
@@ -153,7 +153,7 @@ static PyMethodDef pattern_methods[] = {
 };
 
 PyTypeObject PycairoPattern_Type = {
-  PyVarObject_HEAD_INIT(&PyType_Type, 0)
+  PyVarObject_HEAD_INIT(NULL, 0)
   //PyObject_HEAD_INIT(NULL)
   //0,                                  /* ob_size */
   "cairo.Pattern",                    /* tp_name */
@@ -223,7 +223,7 @@ static PyMethodDef solid_pattern_methods
 };
 
 PyTypeObject PycairoSolidPattern_Type = {
-  PyVarObject_HEAD_INIT(&PyType_Type, 0)
+  PyVarObject_HEAD_INIT(NULL, 0)
   //PyObject_HEAD_INIT(NULL)
   //0,                                  /* ob_size */
   "cairo.SolidPattern",               /* tp_name */
@@ -316,7 +316,7 @@ static PyMethodDef surface_pattern_metho
 };
 
 PyTypeObject PycairoSurfacePattern_Type = {
-  PyVarObject_HEAD_INIT(&PyType_Type, 0)
+  PyVarObject_HEAD_INIT(NULL, 0)
   //PyObject_HEAD_INIT(NULL)
   //0,                                  /* ob_size */
   "cairo.SurfacePattern",             /* tp_name */
@@ -402,7 +402,7 @@ static PyMethodDef gradient_methods[] =
 };
 
 PyTypeObject PycairoGradient_Type = {
-  PyVarObject_HEAD_INIT(&PyType_Type, 0)
+  PyVarObject_HEAD_INIT(NULL, 0)
   //PyObject_HEAD_INIT(NULL)
   //0,                                  /* ob_size */
   "cairo.Gradient",                   /* tp_name */
@@ -473,7 +473,7 @@ static PyMethodDef linear_gradient_metho
 };
 
 PyTypeObject PycairoLinearGradient_Type = {
-  PyVarObject_HEAD_INIT(&PyType_Type, 0)
+  PyVarObject_HEAD_INIT(NULL, 0)
   //PyObject_HEAD_INIT(NULL)
   //0,                                  /* ob_size */
   "cairo.LinearGradient",             /* tp_name */
@@ -546,7 +546,7 @@ static PyMethodDef radial_gradient_metho
 };
 
 PyTypeObject PycairoRadialGradient_Type = {
-  PyVarObject_HEAD_INIT(&PyType_Type, 0)
+  PyVarObject_HEAD_INIT(NULL, 0)
   //PyObject_HEAD_INIT(NULL)
   //0,                                  /* ob_size */
   "cairo.RadialGradient",             /* tp_name */
--- origsrc/pycairo-1.8.10/src/surface.c	2010-09-12 04:51:40.000000000 -0500
+++ src/pycairo-1.8.10/src/surface.c	2010-11-03 15:41:38.954191100 -0500
@@ -346,7 +346,7 @@ static PyMethodDef surface_methods[] = {
 
 
 PyTypeObject PycairoSurface_Type = {
-  PyVarObject_HEAD_INIT(&PyType_Type, 0)
+  PyVarObject_HEAD_INIT(NULL, 0)
   //PyObject_HEAD_INIT(NULL)
   //0,                                  /* ob_size */
   "cairo.Surface",                    /* tp_name */
@@ -646,7 +646,7 @@ static PyMethodDef image_surface_methods
 
 
 PyTypeObject PycairoImageSurface_Type = {
-  PyVarObject_HEAD_INIT(&PyType_Type, 0)
+  PyVarObject_HEAD_INIT(NULL, 0)
   //  PyObject_HEAD_INIT(NULL)
   //0,                                  /* ob_size */
   "cairo.ImageSurface",               /* tp_name */
@@ -772,7 +772,7 @@ static PyMethodDef pdf_surface_methods[]
 };
 
 PyTypeObject PycairoPDFSurface_Type = {
-  PyVarObject_HEAD_INIT(&PyType_Type, 0)
+  PyVarObject_HEAD_INIT(NULL, 0)
   //  PyObject_HEAD_INIT(NULL)
   //0,                                  /* ob_size */
   "cairo.PDFSurface",                 /* tp_name */
@@ -976,7 +976,7 @@ static PyMethodDef ps_surface_methods[]
 };
 
 PyTypeObject PycairoPSSurface_Type = {
-  PyVarObject_HEAD_INIT(&PyType_Type, 0)
+  PyVarObject_HEAD_INIT(NULL, 0)
   //PyObject_HEAD_INIT(NULL)
   //0,                                  /* ob_size */
   "cairo.PSSurface",                  /* tp_name */
@@ -1092,7 +1092,7 @@ static PyMethodDef svg_surface_methods[]
 };
 
 PyTypeObject PycairoSVGSurface_Type = {
-  PyVarObject_HEAD_INIT(&PyType_Type, 0)
+  PyVarObject_HEAD_INIT(NULL, 0)
   //PyObject_HEAD_INIT(NULL)
   //0,                                  /* ob_size */
   "cairo.SVGSurface",                 /* tp_name */
@@ -1158,7 +1158,7 @@ static PyMethodDef win32_surface_methods
 };
 
 PyTypeObject PycairoWin32Surface_Type = {
-  PyVarObject_HEAD_INIT(&PyType_Type, 0)
+  PyVarObject_HEAD_INIT(NULL, 0)
   //PyObject_HEAD_INIT(NULL)
   //0,                                  /* ob_size */
   "cairo.Win32Surface",               /* tp_name */
@@ -1221,7 +1221,7 @@ static PyMethodDef win32_printing_surfac
 };
 
 PyTypeObject PycairoWin32PrintingSurface_Type = {
-  PyVarObject_HEAD_INIT(&PyType_Type, 0)
+  PyVarObject_HEAD_INIT(NULL, 0)
   //PyObject_HEAD_INIT(NULL)
   //0,                                  /* ob_size */
   "cairo.Win32PrintingSurface",       /* tp_name */
@@ -1371,7 +1371,7 @@ static PyMethodDef xcb_surface_methods[]
 };
 
 PyTypeObject PycairoXCBSurface_Type = {
-  PyVarObject_HEAD_INIT(&PyType_Type, 0)
+  PyVarObject_HEAD_INIT(NULL, 0)
   //PyObject_HEAD_INIT(NULL)
   //0,                                  /* ob_size */
   "cairo.XCBSurface",                 /* tp_name */
@@ -1452,7 +1452,7 @@ static PyMethodDef xlib_surface_methods[
 };
 
 PyTypeObject PycairoXlibSurface_Type = {
-  PyVarObject_HEAD_INIT(&PyType_Type, 0)
+  PyVarObject_HEAD_INIT(NULL, 0)
   //PyObject_HEAD_INIT(NULL)
   //0,                                  /* ob_size */
   "cairo.XlibSurface",                /* tp_name */
